05_stringer/undrewb/main_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	want := strconv.Quote("127.0.0.1\n")
+	got := strconv.Quote(buf.String())
```

Comment:
Why `strconv.Quote`?

Reply:
Actually, this seems to be a common pattern among go-course PRs. Is it recommended somewhere to do it this way?

Reply:
It saves you typing `"\"127.0.0.1\n\"" - which is the output of main, which needs to be tested for coverage. that's all.